src/common/TrackedOp.cc

```diff
@@ -278,6 +278,11 @@ void OpTracker::mark_event(TrackedOp *op, const string &dest, utime_t time)
 {
   if (!tracking_enabled)
     return;
+ 
```

Comment:
@joemario Please keep PR title < 50 characters for future PRs, also prepend component name before PR in PR title.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits